apps/ledger-live-desktop/src/renderer/components/PostOnboardingHub/PostOnboardingActionRow.tsx

```diff
@@ -3,38 +3,56 @@ import { Flex, Icons, Tag, Text } from "@ledgerhq/react-ui";
 import { useTranslation } from "react-i18next";
 import { useHistory } from "react-router-dom";
 import { PostOnboardingActionState, PostOnboardingAction } from "@ledgerhq/types-live";
+import styled from "styled-components";
 
 export type Props = PostOnboardingAction & PostOnboardingActionState;
 
+const ActionRowWrapper = styled(Flex)<{ completed: boolean }>`
+  cursor: ${p => (p.completed ? "auto" : "pointer")};
+`;
+
 const PostOnboardingActionRow: React.FC<Props> = props => {
   const {
     navigationParams,
     Icon,
     title,
     description,
     tagLabel,
-    startEvent,
+    startAction,
     startEventProperties,
     completed,
   } = props;
   const { t } = useTranslation();
-  /*const navigation = useNavigation();
+  const history = useHistory();
+
+  const isInsidePostOnboardingScreen = history.location.pathname === "/post-onboarding";
 
-  const handlePress = useCallback(() => {
-    if (navigationParams) {
-      navigation.navigate(...navigationParams);
-      startEvent && track(startEvent, startEventProperties);
+  const handleStartAction = useCallback(() => {
+    if (navigationParams) history.push(navigationParams);
+    else if (startAction) {
+      if (isInsidePostOnboardingScreen) {
+        history.push("/");
+      }
+      startAction();
     }
-  }, [navigationParams, navigation, startEvent, startEventProperties]);*/
+  }, [history, isInsidePostOnboardingScreen, navigationParams, startAction]);
 
   return (
-    <Flex
+    <ActionRowWrapper
       flexDirection="row"
       alignItems="center"
       backgroundColor="neutral.c30"
       borderRadius={3}
       marginBottom={4}
+      completed={completed}
       padding="32px 24px 32px 24px"
+      {...(!completed
+        ? {
+            onClick: () => {
+              handleStartAction();
+            },
+          }
+        : {})}
```

Comment:
Here is it possible to do `onClick={completed ? undefined : handleStartAction}` ?